tb_pulumi/ec2.py

```diff
@@ -71,11 +71,14 @@ def __init__(
     ):
         super().__init__('tb:ec2:NetworkLoadBalancer', name, project, opts=opts)
 
+        # The primary_subnet is just the first subnet listed, used for determining VPC placement
+        primary_subnet = subnets[0]
+
         # Build a security group that allows ingress on our listener port
         security_group_with_rules = tb_pulumi.network.SecurityGroupWithRules(
             f'{name}-sg',
             project,
-            vpc_id=subnets[0].vpc_id,
+            vpc_id=primary_subnet.vpc_id,
```

Comment:
This change comes from feedback from @MelissaAutumn about referencing this by ID constantly instead of having a more representative name for the value.